### ISSUE TITLE
Regional right ventricular hypokinesis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7597]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://pubmed.ncbi.nlm.nih.gov/16549640/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
McConnell’s Sign
https://www.ahajournals.org/doi/10.1161/circulationaha.107.746602

**Comment by pnrobinson** [comment]
added
